src/config/dynamic_content.cc

```diff
@@ -83,7 +83,7 @@ std::shared_ptr<DynamicContent> DynamicContentList::get(std::size_t id, bool edi
 std::shared_ptr<DynamicContent> DynamicContentList::get(const fs::path& location)
 {
     AutoLock lock(mutex);
-    auto entry = std::find_if(list.begin(), list.end(), [&](auto&& c) { return c->getLocation() == location; });
+    auto entry = std::find_if(list.begin(), list.end(), [=](auto&& c) { return c->getLocation() == location; });
```

Comment:
Doesn't this copy `location` or is there some special treatment for const&?

Reply:
There's a size reduction. I don't know why. All the lambdas that resulted in size increases I left alone.

Reply:
Size does not mean that it's better at runtime. I assume the the copy constructor for path is called here, which is linked from stdc++ instead of passing the reference.